packages/odyssey-react/src/components/Field/Field.theme.ts

```diff
@@ -16,11 +16,14 @@ export const theme: ThemeReducer = (theme) => ({
   BorderColor: "currentcolor",
   BorderStyle: "none",
   BorderWidth: 0,
-  FontMaxWidth: theme.FontLineLengthMax,
-  SpaceMarginBlockEnd: theme.SpaceRemM,
-  SpaceMarginBlockStart: 0,
-  SpaceMarginInline: 0,
-  SpacePaddingBlockEnd: theme.SpaceRemM,
-  SpacePaddingBlockStart: 0,
-  SpacePaddingInline: 0,
+  MaxWidth: theme.FontLineLengthMax,
+  MarginBlockEnd: theme.SpaceRemM,
+  MarginBlockStart: 0,
```

Comment:
It feels a bit odd to offer theming for what were previously browser normalization styles. However, our entirely bespoke components (those without browser element styles) offer the same flexibility.

Reply:
@edburyenegren-okta since these are part of what SIWNext wanted I think we'll just focus on giving the people what they want for now. But you're thinking about this the right way, some things we should not expose for theming and we still need a build a way, ideally without SASS, to do that!